lint-manifest.yml

```diff
@@ -0,0 +1,17 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: 18fgsa/concourse-task
+
+inputs:
+- name: pipeline-config
+- name: lint-manifest
+
+params:
+  - MANIFEST_FILE: manifest.yml
+  - LINTER_CONFIG: bosh-lint.yml
```

Comment:
>It's unintuitive that you have to provide a configuration input (because concourse will require it) even if you don't want to override the linter config.

@cnelson will concourse still require config inputs if they have a default value set?